Misc/NEWS.d/next/Core and Builtins/2021-11-22-13-05-32.bpo-45292.pfEouJ.rst

```diff
@@ -0,0 +1 @@
+Complete the :pep:`654` implementation: add ``except*``.
```

Comment:
I created a new news file because the one from the previous PRs is now part of 3.11.0a1 and this one will be in the next tag.